### PR TITLE
Update Mnubo Node-red nodes

### PR DESCRIPTION
**Pull request by DiegoAntonino**
- Adding 'Exponential backoff retries' features.
- Small Changes on NODE information.

### CODE REVIEW

**Comment by lmcjerl** [comment]
This seem OK with me, will wait a bit to see if other have comments, can you give me a quick demo?

**Comment by daddykotex** [comment]
LGTM too @jtheoof

File: nodes/mnubo/config/mnubo-config.html [hunk 1]
```diff
@@ -68,10 +91,15 @@
...
    <ul>
-      <li><p><b>sandbox:</b>to connect to the mnubo's sandbox system</p></li>
-      <li><p><b>production:</b>to connect to the mnubo's production system</p></li>
-      <li><p><b>use Proxy URL:</b>This is optional, if you need to connect to an alternate system, or you need to go thru a proxy, you can use this field, it need to be a valid format (http://hostname:port)</p></li>
+      <li><b>sandbox:</b> to connect to the mnubo's sandbox system</li>
+      <li><b>production:</b> to connect to the mnubo's production system</li>
+      <li><b>use Proxy URL:</b> This is optional, if you need to connect to an alternate system, or you need to go through a proxy. It need to be a valid format (http://hostname:port)</li>
+   </ul>
+   <p><b>Client ID</b> need to be a valid Client ID</p>
+   <p><b>Client Secret</b> need to be the valid associated Secret</p>
```
Comment (jtheoof) [inline nodes/mnubo/config/mnubo-config.html]: needs

File: nodes/mnubo/config/mnubo-config.html [hunk 2]
```diff
@@ -68,10 +91,15 @@
    <p>This is a Mnubo configuration node</p>
    <p><b>Environment</b> can be:</p>
    <ul>
-      <li><p><b>sandbox:</b>to connect to the mnubo's sandbox system</p></li>
-      <li><p><b>production:</b>to connect to the mnubo's production system</p></li>
-      <li><p><b>use Proxy URL:</b>This is optional, if you need to connect to an alternate system, or you need to go thru a proxy, you can use this field, it need to be a valid format (http://hostname:port)</p></li>
+      <li><b>sandbox:</b> to connect to the mnubo's sandbox system</li>
+      <li><b>production:</b> to connect to the mnubo's production system</li>
+      <li><b>use Proxy URL:</b> This is optional, if you need to connect to an alternate system, or you need to go through a proxy. It need to be a valid format (http://hostname:port)</li>
```
Comment (DiegoAntonino) [inline nodes/mnubo/config/mnubo-config.html]: Done

File: nodes/mnubo/config/mnubo-config.html (hunk 1 above)
Comment (DiegoAntonino) [inline nodes/mnubo/config/mnubo-config.html]: done

**Pull request merged**